Localization/Resources/de-DE/winget.resw

```diff
@@ -702,7 +702,7 @@ Sie können über die Einstellungsdatei „winget settings“ konfiguriert werde
     <value>Zeigt verfügbare Upgrades an und führt sie aus.</value>
   </data>
   <data name="Usage" xml:space="preserve">
-    <value>Verwendung: {0}{1}</value>
+    <value>Verwendung: {0} {1}</value>
     <comment>{Locked="{0}","{1}"} Message displayed to provide the user with instructions on how to use a command. {0} is a placeholder replaced by the program name (e.g. 'winget'). {1} is a placeholder replaced by the pattern for using the selected command.</comment>
```

Comment:
Actually. this issue could be fixed by updating locked words in comments in this file [here](https://github.com/microsoft/winget-cli/blob/7b6f58ac20aeaa61444bf4abeb05326ac2b33455/src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw#L714)

to be {Locked="{0} {1}"}

@Speedfighter, would you make that change since you found this issue? Or I can make that change if you are not comfortable making it.

Reply:
Same for this line [here ](https://github.com/microsoft/winget-cli/blob/7b6f58ac20aeaa61444bf4abeb05326ac2b33455/src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw#L845)  and [here](https://github.com/microsoft/winget-cli/blob/7b6f58ac20aeaa61444bf4abeb05326ac2b33455/src/AppInstallerCLIPackage/Shared/Strings/en-us/winget.resw#L1019)

Reply:
I can make these changes when I'm at home.